go/cmd/betabot/main.go

```diff
@@ -57,64 +62,326 @@ func betabot() error {
 		qrterminal.GenerateHalfBlock(res.HTMLURL, qrterminal.L, os.Stdout)
 	}
 
+	var wg sync.WaitGroup
+	// request contact in flag
+	{
+		if *contact != "" {
+			splits := strings.Split(*contact, ",")
+			wg.Add(1)
+			go func() {
```

Comment:
no need for async here, the protocol will be async, and by removing the go routine, you can simplify this part of the code by removing the cancel() and by handling the errors directly as initialization errors